src/app/core/data/data.service.ts

```diff
@@ -45,11 +45,14 @@ export abstract class DataService<T extends CacheableObject> {
   protected abstract store: Store<CoreState>;
   protected abstract linkPath: string;
   protected abstract halService: HALEndpointService;
-  protected abstract forceBypassCache = false;
   protected abstract objectCache: ObjectCacheService;
   protected abstract notificationsService: NotificationsService;
   protected abstract http: HttpClient;
   protected abstract comparator: ChangeAnalyzer<T>;
+  /**
+   * Allows subclasses to reset the response cache time.
+   */
+  protected resetMsToLive: number;
```

Comment:
I think it's a useful property to have on all dataservices, but I'd also call it `responseMsToLive` or perhaps `defaultResponseMsToLive` because that's what it is: the time in ms responses for this dataservice should stay in the cache. 
With the name `resetMsToLive` I would expect a boolean that would set `responseMsToLive` to 0 when `true`.

Reply:
You're right. I originally added a boolean property and then decided that number is more useful here. The name should have changed, too.